addons/website/views/snippets/s_numbers.xml

```diff
@@ -4,32 +4,28 @@
 <template name="Numbers" id="s_numbers">
     <section class="s_numbers o_cc o_cc1 pt80 pb80">
         <div class="container">
-            <div class="row">
-                <div class="col-lg-4">
+            <div class="row s_nb_column_fixed">
```

Comment:
Maybe move this one-line alone in its own PR, for 18.0. The rest we cannot do in 18.0 but this class addition can be done to improve the current version.

Reply:
Here is the PR 

https://github.com/odoo/odoo/pull/185895